ast/Trees.h

```diff
@@ -675,6 +675,11 @@ class EmptyTree final : public Expression {
  * Desugar string concatenation into series of .to_s calls and string concatenations
  */
 
+// TODO Find a better place for this
+bool classDefinesBehavior(std::unique_ptr<ast::ClassDef> &);
+bool ignoreChild(ast::Expression *expr);
```

Comment:
given that this now a public API, let's:
 - give it a descriptive name
- make it take `const std::unique_ptr<ast::ClassDef> &`